pulp_rpm/app/models/modulemd.py

```diff
@@ -44,6 +44,7 @@ class Modulemd(Content):
     context = models.CharField(max_length=255)
     arch = models.CharField(max_length=255)
 
+    static_context = models.BooleanField(default=False)
```

Comment:
static_context is not a part of module's unique key because if the static_context field is added to the existing module, the module's version bumps? Correct? (I think and hope so but I'd like to make sure we cover it here.)
So there can be no module with the same NSVCA and different static_context?

Reply:
I believe that is correct. My reading is that static_context is a note to the build-system to NOT generate a context on module-rebuild. NSVCA is still the module-identifier.